src/commands/basemaps-github/create-pr.ts

```diff
@@ -80,9 +80,20 @@ async function parseVectorTargetInfo(target: string): Promise<{ name: string; ti
 
   if (epsg == null || name == null) throw new Error(`Invalid target ${target} to parse the epsg and imagery name.`);
   if (epsg !== Epsg.Google) throw new Error(`Unsupported epsg code ${epsg.code} for vector map.`);
-  // Try to get the region for individual layers
+  // Try to get the title
+  const collectionPath = fsa.join(target, 'collection.json');
+  const collection = await fsa.readJson<StacCollection>(collectionPath);
+  if (collection == null) throw new Error(`Failed to get target collection json from ${collectionPath}.`);
+  const ldsLayers = collection.links.filter((f) => f.rel === 'lds:layer');
+  let title = collection.title;
+  // Get title from lds:title for individual vector layer
+  if (ldsLayers.length === 1) {
```

Comment:
Why only if there's exactly one layer? Would it be wrong to get the title from the first layer if there's more than one layer?

Reply:
We should only get the lds title for individual layer. Otherwise, for the vector map that have a mixed of multiple lds layers, we can use the `collection.title`.

Reply:
OIC, if there's exactly one layer its title is preferred over the collection title. Strange, but sure.